Source/Core/VideoCommon/PixelShaderGen.cpp

```diff
@@ -240,9 +237,12 @@ PixelShaderUid GetPixelShaderUid()
   {
     uid_data->stagehash[n].tevorders_texcoord = bpmem.tevorders[n / 2].getTexCoord(n & 1);
 
+    // hasindstage previously was used as a criterion to set tevind to 0, but there are variables in
+    // tevind that are used even if the indirect stage is disabled, so now it is only left in to
+    // avoid breaking existing UIDs (in most cases, games will have 0 in tevind anyways)
+    // TODO: Remove hasindstage on the next UID version bump
```

Comment:
Wondering if this TODO should be next to the UID with a reference to this location?

Reply:
Good idea, done.